src/styles/components/generic/_grid.scss

```
@@ -0,0 +1,71 @@
+$viewports: (
+    sm: 480,
+    md: 780,
+    lg: 1024,
+);
+
+@mixin makeGrid {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-column-gap: .75rem;
```

Stylelint (17.12.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)